OOStepByStepTest/UnitTest1.cs

```diff
@@ -1,15 +1,96 @@
 namespace OOStepByStepTest
 {
     using OOStepByStep;
+    using System;
     using Xunit;
 
     public class UnitTest1
     {
-        [Fact]
-        public void Test1()
+        [Theory]
+        [InlineData("haha", 11)]
```

Comment:
try to use name and age existing in requirement.
since they are closer to the business scenario.

Reply:
fixed!